squashfs-tools/pkg.yaml

```diff
@@ -0,0 +1,7 @@
+name: squashfs-tools
+variant: scratch
+dependencies:
+  - image: "{{ .TOOLS_PREFIX }}tools-squashfs-tools:{{ .TOOLS_REV }}"
+finalize:
+- from: /
```

Comment:
Please copy via `/rootfs` as I did in #1161 to avoid /pkg in the output in all pkgs, to not copy from /